chai-dom.js

```diff
@@ -355,4 +355,17 @@
       , actual
     )
   })
+
+  chai.Assertion.addMethod('style', function (styleProp, styleValue) {
```

Comment:
I like this. Sorry for the delay; can you add documentation?

Reply:
@nathanboktae sure thing, I've updated README.md. Thanks for keeping the repo maintained!